kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/KieServerConstants.java

```diff
@@ -33,6 +33,7 @@
     public static final String KIE_DMN_SERVER_EXT_DISABLED = "org.kie.dmn.server.ext.disabled";
     public static final String KIE_SWAGGER_SERVER_EXT_DISABLED = "org.kie.swagger.server.ext.disabled";
     public static final String KIE_PROMETHEUS_SERVER_EXT_DISABLED = "org.kie.prometheus.server.ext.disabled";
+    public static final String KIE_SCENARIO_SIMULATION_SERVER_EXT_DISABLED = "org.kie.scenariosimulation.server.ext.disabled";
```

Comment:
@danielezonca could you please open aslo PR for kiegroup/kie-docs explaining this property?

Reply:
Created [this ticket](https://issues.redhat.com/browse/DROOLS-4883) to do it